scripts/monitor/notebook/plot-cromwell-bq-monitoring.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Do you think this should also be saved in the pickle? Reason: users may not want to recreate this every time.</p><p> </p><p>A better fix will be to move this into the class definition itself.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/long-read-pipelines/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I don't see any problem with doing that.</p>